src/main.c

```diff
@@ -518,6 +589,12 @@ int main(int argc, char *argv[])
 
 		fprintf(out, "%s (%s)\n", err ?: "-", errmsg ?: "-");
 	}
+	else if (ret == ERR_USBLOOP)
+	{
+		fputs("\nError: ev3duder received its own packet instead of a reply from the brick.\n\n", stderr);
+		fputs("If you have plugged the EV3 into a USB 3.0 port (usually red or blue),\n", stderr);
+		fputs("try plugging it into a USB 2.0 only port (usually black or white).\n", stderr);
```

Comment:
Huh, that's stange. I would have thought that wouldn't matter if the brick doesn't indicate USB 3.0 support per strapping the line.

Reply:
Yup, the USB communication still has to be 2.0, but something on one side of the channel has to contain a bug that triggers this behaviour. However, I cannot point with certainty at the host or at the device, as I don't know what is really transmitted through the bus, only what the Linux receives from its drivers.